packages/@okta/i18n/src/properties/login.properties

```diff
@@ -979,6 +979,9 @@ oie.registration.is.not.enabled = Sign up is not enabled for this organization.
 oie.forgot.password.is.not.enabled = Forgot password is not enabled for this organization.
 idx.return.error = Could not process this email link. Return to the screen where you requested it.
 idx.return.stale = This email link is out of date. Return to the screen where you requested it.
+idx.invalid.forgot.password.token = Your reset password token is no longer valid.
+idx.invalid.reset.password.token = Your reset password token is no longer valid. Contact your administrator to obtain a new token.
+idx.reset.password.activating.user = Reset password cannot be performed while the user is activating. Contact your administrator to obtain a new token.
```

Comment:
* These two errors have an extra space ` ` character in between the two sentences in the monolith.
* I have removed the extra space and made it a single space in-between the two sentences.
* I haven't come across any selenium tests that check for these yet, so I don't expect this to break anything ATM.